ar_hardware_interface/src/ar_hardware_interface.cpp

```diff
@@ -131,6 +138,18 @@ hardware_interface::return_type ARHardwareInterface::write(
   }
   RCLCPP_DEBUG_THROTTLE(logger_, clock_, 500, logInfo.c_str());
   driver_.update(actuator_commands_, actuator_positions_);
+  if (driver_.isEStopped()) {
+    std::string logWarn = "Hardware in EStop state. To reset the EStop "
```

Comment:
For convenience, we could put the 3 commands in a bash script in src/ar_hardware_interface/scripts and do something like `ros2 run ar_hardware_interface reset_estop.sh`

Reply:
This is a great idea! I added a script to do that.

I also added some notes about using EStop (and resetting it) in the readme. Feel free to edit/remove those changes as you see fit